openjdk-test-common/src/test/resources/workspace/setup-java.bash

```diff
@@ -102,7 +103,7 @@ if [ "$TEST_MIN_HEAP" != "OK" -o "$TEST_MAX_HEAP" != "OK" ]; then
 fi
 
 
-#test GKE envrionment
+#test GKE environment
```

Comment:
there are a bunch of places where this file unsets all the parameters, that should be extracted to a function + we should include the `PROFILER_AGENT` variable in it as well.